xmhf/src/xmhf-core/xmhf-runtime/xmhf-eventhub/arch/x86/vmx/peh-x86vmx-main.c

```diff
@@ -682,10 +682,14 @@ static void _vmx_handle_intercept_eptviolation(VCPU *vcpu, struct regs *r){
 	if(vcpu->isbsp && (gpa >= g_vmx_lapic_base) && (gpa < (g_vmx_lapic_base + PAGE_SIZE_4K)) ){
 		xmhf_smpguest_arch_x86_eventhandler_hwpgtblviolation(vcpu, (u32)gpa, errorcode);
 	}else{ //no, pass it to hypapp
-		xmhf_smpguest_arch_x86vmx_quiesce(vcpu);
+		// [Superymk] Some hypapps cannot use CPU quiescing when handling trapped PIO and memory accesses. For example, some
+		// hypapps must call another core to emulate the trapped CPU instructions. These hypapps cannot do so if CPU 
+		// quiescing is used.
+		// [TODO] Add a compiler flag to select this preference.
```

Comment:
Can we add the compiler flag in this PR instead of leaving it for future work? I think TrustVisor may get into trouble because it extensively uses EPT violations to switch between PALs and normal code.

Reply:
Sure, will do